plugin.xml

```diff
@@ -200,4 +200,22 @@
 
 
     </platform>
+    
+    <platform name="browser">
+
+        <config-file target="config.xml" parent="/*">
+            <feature name="Mixpanel">
+                <param name="browser-package" value="MixpanelPlugin"/>
+            </feature>
+        </config-file>
+
+        <js-module src="src/browser/MixpanelProxy.js" name="MixpanelProxy">
```

Comment:
There is 2 distinct things : the snippet and the lib. 
You can check this article : https://mixpanel.com/blog/2014/08/18/integrating-mixpanel-with-cordova
The snippet loads the lib. 
But I've modified the snippet to load the lib locally (this is also explained in their blog article).